pkg/xray/resource_xray_repository_config.go

```diff
@@ -0,0 +1,351 @@
+package xray
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jfrog/terraform-provider-shared/util"
+	"github.com/jfrog/terraform-provider-shared/validator"
+)
+
+type RepoConfig struct {
```

Comment:
Not updating this to `RepoConfiguration`?

Reply:
Updated.